blocks/library/latest-comments/style.scss

```diff
@@ -0,0 +1,23 @@
+
+.editor-visual-editor__block[data-type="core/latest-comments"] {
+
+	.wp-block-latest-comments {
+		padding-left: 2.5em;
+	}
+
+	.wp-block-latest-comments__comment-avatar {
+		display: block;
+		float: left;
+		height: 48px;
+		width: 48px;
+		border-radius: 24px;
+		margin-right: 5px;
```

Comment:
There are 30 instances of `em` units being used in block CSS, whereas there are 37 instances of `px` units being used. So I'd say `px` is safe. 👍